packages/core/useWebWorkerFn/index.stories.tsx

```diff
@@ -1,11 +1,14 @@
 import dayjs from 'dayjs'
 import { defineDemo, html } from '../../_docs'
-import { defineComponent, computed } from 'vue-demi'
+import { defineComponent, computed, ref } from 'vue-demi'
 import { useWebWorkerFn, WorkerStatus } from '.'
 import { useNow } from '../useNow'
 
-const numbers: number[] = [...Array(5000000)].map(_ => ~~(Math.random() * 1000000))
-const sortNumbers = (nums: number[]): number[] => nums.sort()
+const heavyTask = () => {
+  const randomNumber = () => ~~(Math.random() * 5000000)
+  const numbers: number[] = Array(randomNumber()).fill(undefined).map(randomNumber)
```

Comment:
Should we make the Array size fixed to the blocking time could be more consistent?